sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/OrderbyDocumentQueryTest.java

```diff
@@ -488,6 +542,25 @@ public void before_OrderbyDocumentQueryTest() throws Exception {
             props.put("propArray", orderByArray);
             props.put("propObject", orderByObject);
             keyValuePropsList.add(props);
+            switch (i % 5) {
```

Comment:
> switch (i % 5) { [](start = 11, length = 17)

missing null and undefined.

Reply:
undefined is include. There are some documents being inserted above for which propMixed is undefined.